packages/cpt-ui/src/components/prescriptionDetails/ItemsCards.tsx

```diff
@@ -75,9 +75,17 @@ export function ItemsCards({items}: ItemsProps) {
               {dosageInstructions && (
                 <SummaryListRow label={STRINGS.INSTRUCTIONS_LABEL} value={dosageInstructions}/>
               )}
-
               {pharmacyStatus && (
-                <SummaryListRow label={STRINGS.PHARMACY_STATUS_LABEL} value={pharmacyStatus}/>
+                <SummaryListRow
```

Comment:
If we make the change the the SummaryListRow component above can we then update this to just be something like:
```suggestion
                {pharmacyStatus && (
                <SummaryListRow
                  label={STRINGS.PSU_STATUS_LABEL}
                  tagValue={pharmacyStatus}
                  />
              )}
```